gateway-protocol-impl/src/main/resources/gateway-service-config.json

```diff
@@ -0,0 +1,50 @@
+{
+  "methodConfig": [
+    {
+      "name": [
+        {"service": "gateway_protocol.Gateway", "method": "ActivateJobs"},
+        {"service": "gateway_protocol.Gateway", "method": "BroadcastSignal"},
+        {"service": "gateway_protocol.Gateway", "method": "CancelProcessInstance"},
+        {"service": "gateway_protocol.Gateway", "method": "CompleteJob"},
+        {"service": "gateway_protocol.Gateway", "method": "DeleteResource"},
+        {"service": "gateway_protocol.Gateway", "method": "EvaluateDecision"},
+        {"service": "gateway_protocol.Gateway", "method": "FailJob"},
+        {"service": "gateway_protocol.Gateway", "method": "ResolveIncident"},
+        {"service": "gateway_protocol.Gateway", "method": "SetVariables"},
+        {"service": "gateway_protocol.Gateway", "method": "StreamActivatedJobs"},
+        {"service": "gateway_protocol.Gateway", "method": "ThrowError"},
+        {"service": "gateway_protocol.Gateway", "method": "Topology"},
+        {"service": "gateway_protocol.Gateway", "method": "UpdateJobRetries"},
+        {"service": "gateway_protocol.Gateway", "method": "UpdateJobTimeout"}
+      ],
+      "waitForReady": true,
+      "retryPolicy": {
+        "maxAttempts": 5.0,
+        "initialBackoff": "0.1s",
+        "maxBackoff": "5s",
+        "backoffMultiplier": 3.0,
+        "retryableStatusCodes": ["UNAVAILABLE", "RESOURCE_EXHAUSTED", "DEADLINE_EXCEEDED"]
```

Comment:
❌ `DEADLINE_EXCEEDED` is not a retryable status code for:
- `BroadcastSignal`
- `ThrowError`
- `UpdateJobRetries`
- `UpdateJobtimeout`

These are not idempotent. Similar to `PublishMessage` and such below.